Modules/modExtendSaveAsText.ACM

```diff
@@ -72,8 +72,12 @@ Public Sub SaveAsText(objType As asObjectType, objName As String, strFilename As
         MsgBox "Something went wrong cleaning the file for " & objName & "."
       End If
       If FileExists(strFilename & ".tmp") Then Kill strFilename & ".tmp"
-    Case asObjectType.acQuery, asObjectType.acMacro, asObjectType.acModule
+      If Forms("Access SVN - Options").chkOutputUTF8 And IsUCS2(strFilename) Then ConvFileUCS2_to_UTF8 strFilename 'convert file to UTF8

```

Comment:
If the old file was UTF 8 should we overide the `chkOutputUTF8` option? Or at least warn the user?

Reply:
In  reflection I think we should override and I have implemented this logic.